CMakeLists.txt

```diff
@@ -160,6 +166,11 @@ if(ES_GLES)
 	target_link_libraries(EndlessSky PRIVATE OpenGL::OpenGL OpenGL::EGL)
 	target_compile_definitions(EndlessSkyLib PUBLIC ES_GLES)
 else()
+	# The Steam runtime can't rely on GLVND being available, so link against the legacy interface.
+	if(ES_STEAM)
+		set(OpenGL_GL_PREFERENCE "LEGACY")
```

Comment:
This seems like something to cause headaches down the line, no?

Reply:
Why would it?

Reply:
Using a legacy API? Or do we just link against the legacy interface, and at runtime we'll have an uptodate but ABI-compatible implementation?

Reply:
It's legacy but it's still widely available and I'm not sure if it will ever be removed. Some older distributions don't ship with the modern one, so it's not guaranteed that the new one is available. 

> Or do we just link against the legacy interface, and at runtime we'll have an uptodate but ABI-compatible implementation?

exactly